app/models/passenger.rb

```diff
@@ -0,0 +1,29 @@
+class Passenger < ApplicationRecord
+  has_many :trips, dependent: :nullify
+
+  def all_trips
+    temp_trips = []
+    all_trips = []
+    temp_trips << Trip.where(passenger_id: id)
+    temp_trips.each do |trips|
+      trips.each do |trip|
+        all_trips << trip
+      end
+    end
+    return all_trips
+  end
```

Comment:
Same as above: Why do we need to build a new method when `Passenger` `has_many` `Trip`s?